samples/browser/src/main/java/org/mozilla/samples/browser/Components.kt

```diff
@@ -17,9 +17,9 @@ import org.mozilla.geckoview.GeckoRuntime
 /**
  * Helper class for lazily instantiating components needed by the application.
  */
-class Components(private val applcationContext: Context) {
+class Components(private val applicationContext: Context) {
     private val geckoRuntime by lazy {
-        GeckoRuntime.create(applcationContext)
+        GeckoRuntime.getDefault(applicationContext)
```

Comment:
That's one option. In Fenix I initialize the Components object from the application object instead from the activity which can be re-created multiple times during the process lifetime.

Reply:
Yes, let's address this one separately. I have more thoughts here after debugging #157, that I'd like to share with you.